diff --git a/src/bun.js/bindings/BunProcess.cpp b/src/bun.js/bindings/BunProcess.cpp
index d3c8cf1..c972517 100644
--- a/src/bun.js/bindings/BunProcess.cpp
+++ b/src/bun.js/bindings/BunProcess.cpp
@@ -172,6 +172,8 @@ static JSValue constructArch(VM& vm, JSObject* processObject)
     return JSC::jsString(vm, makeAtomString("x64"_s));
 #elif CPU(ARM64)
     return JSC::jsString(vm, makeAtomString("arm64"_s));
+#elif CPU(RISCV64)
+    return JSC::jsString(vm, makeAtomString("riscv64"_s));
 #else
 #error "Unknown architecture"
 #endif
@@ -2293,6 +2295,9 @@ JSC::JSValue generateNodeConfigBinding(JSC::JSGlobalObject* globalObject)
 #elif CPU(ARM64)
     variables->putDirect(vm, JSC::Identifier::fromString(vm, "host_arch"_s), JSC::jsString(vm, String("arm64"_s)), 0);
     variables->putDirect(vm, JSC::Identifier::fromString(vm, "target_arch"_s), JSC::jsString(vm, String("arm64"_s)), 0);
+#elif CPU(RISCV64)
+    variables->putDirect(vm, JSC::Identifier::fromString(vm, "host_arch"_s), JSC::jsString(vm, String("riscv64"_s)), 0);
+    variables->putDirect(vm, JSC::Identifier::fromString(vm, "target_arch"_s), JSC::jsString(vm, String("riscv64"_s)), 0);
 #else
 #error "Unsupported architecture"
 #endif
