diff --git a/src/bun.js/bindings/webcore/SerializedScriptValue.cpp b/src/bun.js/bindings/webcore/SerializedScriptValue.cpp
index 3125ee8..64a60d2 100644
--- a/src/bun.js/bindings/webcore/SerializedScriptValue.cpp
+++ b/src/bun.js/bindings/webcore/SerializedScriptValue.cpp
@@ -3424,12 +3424,13 @@ private:
         ptr += length * sizeof(char16_t);
 #else
         std::span<char16_t> characters;
-        str = String::createUninitialized(length, characters);
+        String identifierString = String::createUninitialized(length, characters);
         for (unsigned i = 0; i < length; ++i) {
             uint16_t c;
             readLittleEndian(ptr, end, c);
             characters[i] = c;
         }
+        str = Identifier::fromString(vm, identifierString);
 #endif
         return true;
     }
